syndeo-template/README.md

```diff
@@ -18,6 +18,50 @@ transforms. Then, the template will take the pipeline spec, and
 
 ## Common workflow tasks
 
+### Format code
+
+To apply spotless formatting rules to the Syndeo template code, run the following command:
+
+```shell
+mvn -B spotless:apply compile -f  unified-templates.xml -pl syndeo-template/pom.xml
+```
+
+### Run tests (unit and integration tests)
+
+To **run unit tests** for the Syndeo template, run the following command. Note that this command knows to skip
+integration tests and only runs unit tests:
+
+```shell
+mvn clean package test -f unified-templates.xml -pl syndeo-template/pom.xml
```

Comment:
I ran this command in a Google Cloud Shell after cloning https://github.com/pabloem/DataflowTemplates/tree/bq-2-bt.

```
mvn clean package test -f unified-templates.xml -pl syndeo-template/pom.xml
```

I received the following error:

```
[ERROR] Failed to execute goal on project syndeo-template: Could not resolve dependencies for project com.google.cloud.teleport.syndeo:syndeo-template:jar:1.0-SNAPSHOT: Failure to find com.google.cloud.teleport:integration-testing-lib:jar:1.0-SNAPSHOT in https://repository.apache.org/content/repositories/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of apache.snapshots has elapsed or updates are forced
```

Reply:
this means you'll need to build the integration test project first? Have you tried that? (I don't know how to make sure it gets built first...)